client/core/core.go

```diff
@@ -908,12 +909,13 @@ func (c *Core) CreateWallet(appPW, walletPW []byte, form *WalletForm) error {
 	}
 	encPW, err := crypter.Encrypt(walletPW)
 	if err != nil {
-		return fmt.Errorf("wallet password encryption error: %v", err)
+		return fmt.Errorf("Wallet password encryption error: %v", err)
```

Comment:
It's still not clear to me whether error messages should be capitalized or not.

Reply:
I've been doing not capitalized because some linter says it's right that way, but I'm not sure I have a good justification for that.

Reply:
This one: https://staticcheck.io/docs/checks#ST1005
Dunno if the reasoning is solid though.

Reply:
`golint` too apparently: https://github.com/golang/lint/blob/738671d3881b9731cc63024d5d88cf28db875626/lint.go#L1150-L1151

Reply:
At least it's something to cite. Undone.